docs/guide/execute-transaction.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: splash
+---
+
+# Execute Transaction
+
+`executeTransaction` function can be used to perform many transactions. It supports every transaction which is possible in algorand network. Ex: Deploy ASA/SSC, Opt-In, Transfers, Delete, Destroy etc. `executeTransaction` takes `ExecParams` or `ExecParams[]` as parameter.
+If you pass an array of `ExecParams`, it will be considered as `atomic transaction`.
+In below sections we will demonstrate how to pass these parameters.
+
+Note: For each parameter `type` and `sign` fields are mandatory.
+  - `type`: type of transaction
+  - `sign`: sign type
+
+Examples of parameter(`ExecParams`):
```

Comment:
We should explain common parameters here: `TxParams` (passed in `execParams.payflags`)

Reply:
No I don't think so, because it will be copy pasting doc string of that interface, main goal of this task is to give clear picture on what to pass, because this part is cryptic.

Reply:
after looking that we don't have explanation in api docs https://algobuilder.dev/api/algob/interfaces/runtime.types.txparams.html , we should add more comments on code in types and add a link to that in docs.

Reply:
As you prefer. I just want these explanations to be included in user docs (as user will probably start from here).

Reply:
yes, api docs are incomplete, added docs

Reply:
Let's add a link to API docs in the first line of the section.

Reply:
OK